monitoring.sh

```diff
@@ -28,7 +27,6 @@ fi
 cmake $SOURCEDIR                                              \
       -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                     \
       ${BOOST_REVISION:+-DBOOST_ROOT=$BOOST_ROOT}                 \
-      ${APMON_CPP_REVISION:+-DAPMON_ROOT=$APMON_CPP_ROOT}         \
```

Comment:
This one I guess we can actually leave, no? Or you removed the option completely?

Reply:
Option is still there, I don't have preference so I brought it back